replicate-to-sql-with-generic-events/lib/postgres.js

```diff
@@ -0,0 +1,54 @@
+const { Pool } = require('pg');
+
+const pgConfig = {
```

Comment:
This should be part of the plugin configuration instead, and the password must be either in the Vault or either in an environment variable

Reply:
You are right, i moved all the script inside the docker and modified how tests are run.

Reply:
I still don't understand why you left this here.  

This kind of configuration should be in the `kuzzlerc` file:
```js
{
  // maybe you should rename the plugin with something more concise in manifest.json
  "replicate-to-sql-with-generic-events": {
    "postgresql": {
      "user": 'kuzzle',
      "database": 'kuzzle',
      "port": 5432
    }    
  }
}
```
And usually it's considered a good practice to allow the usage of environment variable prior to the configuration:
```js
const postgreConfig = {
  database: process.env.PG_DATABASE || config.postgresql.database,
  port: process.env.PG_PORT || config.postgresql.port,
  user: process.env.PG_USER || config.postgresql.user,
  // note the vault usage here
  password: process.env.PG_PASSWORD || vault.postgresql.password
};
```
But since this how-to is not intended to be used "as-is", I won't block this PR for that.